libraries/botbuilder-dialogs-adaptive-testing/tests/testTelemetryProperties.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * Return expected properties for codeIntent utterance 'intent a1 b2'.
+ *
+ * @returns {*} Object of expected telemtry properties.
+ */
+const getCodeIntentProperties = () => {
```

Comment:
I'm only leaving this comment as I like to write code that's terse, when possible (and readable).

Returns are implied if you omit braces - i.e. `const foo = () => true` is a function that returns `true`.

I'm not suggesting you need to rewrite these functions, just wanted to mention it!